Pod/Classes/NWSTokenView.swift

```diff
@@ -47,8 +40,33 @@ public class NWSTokenView: UIView, UIScrollViewDelegate, UITextViewDelegate
     
     // MARK: Private Vars
     private var shouldBecomeFirstResponder: Bool = false
-    private var scrollView = UIScrollView()
+    public var scrollView = UIScrollView()
```

Comment:
@wfilleman Whats the necessity of making this public? I'd prefer for users not to be able to change non-public properties on this as it could cause a lot of errors in the way the token view builds and displays the tokens.